@l10n/ja/docs/references/http-websocket-apis/public-api-methods/account-methods/account_info.md

```diff
@@ -253,7 +253,7 @@ rippled account_info rG1QQv2nh2gr7RCZ1P8YYcBUKCCN633jCn validated
 ## 考えられるエラー
 
 * いずれかの[汎用エラータイプ][]。
-* `invalidParams` - 1つ以上のフィールドの指定が正しくないか、1つ以上の必須フィールドが指定されていません。例えば、リクエストで`queue`を`true`と指定したが、現在開いているレジャーではない`ledger_index`を指定した場合です。
+* `invalidParams` - 1つ以上のフィールドの指定が正しくないか、1つ以上の必須フィールドが指定されていません。例えば、リクエストで`queue`を`true`と指定したが、現在開いているレジャーではない`ledger_index`を指定した場合です。[API v1]()では、`signer_lists`に非真偽値を指定した場合にこのエラーが発生します。
```

Comment:
Syntax error here with parentheses instead of brackets.

```suggestion
* `invalidParams` - 1つ以上のフィールドの指定が正しくないか、1つ以上の必須フィールドが指定されていません。例えば、リクエストで`queue`を`true`と指定したが、現在開いているレジャーではない`ledger_index`を指定した場合です。[API v1][]では、`signer_lists`に非真偽値を指定した場合にこのエラーが発生します。
```